assets/base.css

```diff
@@ -3353,6 +3353,11 @@ details-disclosure > details {
     transform: rotate(0.5deg); /* Less intense rotation for collage items. */
   }
 
+  .animate--hover-3d-lift .grid__item:hover,
```

Comment:
applying it to `grid__item` scares me a little. It feels to generic/general. Lots of elements that aren't getting that effect I believe would still be using this class. 
So maybe we could be more specific at targeting containers of elements that do have the 3d lift applied to them 🤔 

A couple ways we could go about it: 
- use a selector that is yet to be supported on all browsers, so the experience isn't the same everywhere but it's a "cute" fix not specifically functional. So using grid__item:hover:has(.card, .collection__card, etc)`
- target something a bit more specific. `collection-list__item`, `.product-grid .grid__item`, `.complementary-slide li`

Reply:
👍 I targeted more specific classes in https://github.com/Shopify/dawn/pull/2589/commits/777fa46e2077d48c9d981c6c0ee5802cfdd22d8e. I _think_ I got them all: product grids, collection lists, collage items, blog articles, and complementary products. Let me know if you can think of any other contexts we should hit.